cmd/cli/kubectl-kyverno/test/test_command.go

```diff
@@ -441,6 +441,7 @@ func testCommandExecute(dirPath []string, fileName string, gitBranch string, tes
 	}
 
 	if rc.Fail > 0 {
```

Comment:
Can we move this block below L448. That way we don't have to duplicate the summary line.

Reply:
sounds cool.....i'll do it:)